source/dialog/startup/startup_start2.sqf

```diff
@@ -0,0 +1,622 @@
+    };
+
+};
+
+// FAST TRAVEL
+_index = lbCurSel 2714;
+switch (_index) do
+{
+
+    case index_fast_travel_false:  
+    {
+    enable_fast_travel = false;
+    };
+
+    case index_fast_travel_true:  
+    {
+    enable_fast_travel = true;
+    };
+
+};
+
+// MAX DIST FROM HQ
+_index = lbCurSel 2188;
+switch (_index) do
+{
+
+    case index_max_dist_hq_20:  
+    {
+    zones_max_dist_from_hq = 2000;
+    };
+	
+	case index_max_dist_hq_25:  
+    {
+    zones_max_dist_from_hq = 2500;
+    };
+	
+	case index_max_dist_hq_30:  
+    {
+    zones_max_dist_from_hq = 3000;
+    };
+	
+	case index_max_dist_hq_40:  
+    {
+    zones_max_dist_from_hq = 4000;
+    };
+
+    case index_max_dist_hq_50:  
+    {
+    zones_max_dist_from_hq = 5000;
+    };
+	
+    case index_max_dist_hq_75:  
+    {
+    zones_max_dist_from_hq = 7500;
+    };
+
+    case index_max_dist_hq_100:  
+    {
+    zones_max_dist_from_hq = 10000;
+    };
+	
+    case index_max_dist_hq_125:  
+    {
+    zones_max_dist_from_hq = 12500;
+    };
+
+    case index_max_dist_hq_150:  
+    {
+    zones_max_dist_from_hq = 15000;
+    };
+	
+    case index_max_dist_hq_175:  
+    {
+    zones_max_dist_from_hq = 17500;
+    };
+
+    case index_max_dist_hq_200:  
+    {
+    zones_max_dist_from_hq = 20000;
+    };
+    case index_max_dist_hq_250:  
+    {
+    zones_max_dist_from_hq = 25000;
+    };
+
+    case index_max_dist_hq_300:  
+    {
+    zones_max_dist_from_hq = 30000;
+    };
+	
+    case index_max_dist_hq_400:  
+    {
+    zones_max_dist_from_hq = 40000;
+    };
+
+    case index_max_dist_hq_500:  
+    {
+    zones_max_dist_from_hq = 50000;
+    };	
+};
+
+if (zones_max_radius <= zones_min_radius) exitWith {Hint "Unable to generate campaign:\nMaximum radius of a zone\nmust be superior to\nMinimum radius of a zone"}; // CHECK IF MAX/MIN RADIUS PARAMETERS CORRECT
+
+zones_spacing = zones_max_radius + 200;
+chosen_settings = true;  //  Give the go ! BluHQinit.sqf can continue execution
+manually_chosen = true;
+publicVariable "chosen_settings";
+publicVariable "commandpointsblu1";
+publicVariable "weather_type";
+publicVariable "blufor_ai_skill";
+publicVariable "opfor_ai_skill";
+publicVariable "enableChopperFastTravel";
+publicVariable "manually_chosen";
+
+	//	commandpointsblu1 = 9999999; 
+//hint format["Max radius: %1\nMin radius: %2\nZones number: %3\nCommand points: %4\nBLU AP: %5\nOPF AP: %6\nWeather type: %7\nBLU AI skill: %8\nOPF AI skill: %9",zones_max_radius,zones_min_radius,zones_number,commandpointsblu1,opfor_ap,blufor_ap,weather_type,blufor_ai_skill,opfor_ai_skill];
+diag_log format["----- DUWS CHOSEN SETTINGS --------- Max radius: %1-------Min radius: %2-------Zones number: %3-------Command points: %4-------BLU AP: %5-------OPF AP: %6-------Weather type: %7-------BLU AI skill: %8-------OPF AI skill: %9",zones_max_radius,zones_min_radius,zones_number,commandpointsblu1,opfor_ap,blufor_ap,weather_type,blufor_ai_skill,opfor_ai_skill];
+closeDialog 0;
```

Comment:
The formatting here is off. Can you please format the cases where the body inside of curly braces is kicked in? Example 1 or 2 is fine.
#### Example 1
```
switch(foo) do
{
    case bar:
    {
        blah command;
    }
};
```
#### Example 2:
```
switch(foo) do {
    case bar: {
        blah command;
    }
};
```